ui/app/styles/core/buttons.scss

```diff
@@ -280,5 +280,4 @@ a.button.disabled {
   font-size: inherit;
   font-weight: inherit;
   cursor: pointer;
-  color: $link;
```

Comment:
I think this was originally intended to be a sort of reset class to make a button look like the surrounding text and then to use additional classes to change the color or underline for specific scenarios. `color: inherit` was already defined on line 279.

Reply:
thanks for cleaning up!